homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,154 @@
+set_manual_level:
+  fields:
+    manual_level:
+      required: true
+      advanced: false
+      default: "Off"
+      selector:
+        select:
+          options:
+            - "Off"
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+            - "Breeze"
+            - "Holiday"
+
+set_timer_level:
+  fields:
+    timer_level:
+      required: true
+      advanced: false
+      default: "Level1"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+            - "Holiday"
+            - "Breeze"
+    time:
+      required: true
+      advanced: false
+      default: 0
+      selector:
+        number:
+          min: 0
+          max: 1440
+          step: 10
+          unit_of_measurement: "min"
+          mode: slider
+
+set_breeze:
+  fields:
+    breeze_level:
+      required: false
+      advanced: false
+      default: "Level3"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+    temperature:
+      required: false
+      advanced: false
+      default: 18
+      selector:
+        number:
+          min: 15
+          max: 35
+          step: 1
+          unit_of_measurement: "°C"
+          mode: slider
+    activate:
+      required: true
+      advanced: false
+      default: false
+      selector:
+        boolean:
+
+set_day_night_time:
+  fields:
+    day:
+      required: true
+      advanced: false
+      default: false
```

Comment:
Please set the default value here to the default value used in the code.
Same for the `night` attribute.